packages/dialtone-vue3/recipes/leftbar/contact_centers_row/contact_centers_row.test.js

```diff
@@ -0,0 +1,87 @@
+import { mount } from '@vue/test-utils';
+import DtRecipeContactCentersRow from './contact_centers_row.vue';
+
+// Constants
+const basePropsData = {
+  description: 'Ai Contact Centers',
+};
+
+describe('DtRecipeContactCentersRow Tests', () => {
+  // Wrappers
+  let wrapper;
+  let iconType;
+  let description;
+  let unreadBadge;
+
+  // Environment
+  let propsData = basePropsData;
+  let attrs = {};
+  let slots = {};
+  let provide = {};
+
+  // Helpers
+  const _setChildWrappers = async () => {
+    await vi.dynamicImportSettled();
+    console.log(wrapper.html());
+    iconType = wrapper.find('[data-qa="dt-leftbar-row-icon"]');
+    description = wrapper.find('[data-qa="dt-leftbar-row-description"]');
+    unreadBadge = wrapper.find('[data-qa="dt-leftbar-row-unread-badge"]');
+  };
+
+  const _setWrappers = async () => {
+    wrapper = mount(DtRecipeContactCentersRow, {
+      propsData,
```

Comment:
in vue 3 this is named `props` only, check https://github.com/dialpad/dialtone/blob/staging/packages/dialtone-vue3/.github/TEST_CONTRIBUTING/TEST_GUIDELINE.md for guidance.